.circleci/config.yml

```diff
@@ -7,16 +7,16 @@ jobs:
             - checkout
             - restore_cache:
                   keys:
-                      - v1-dependencies-{{ checksum "package.json" }}
-                      - v1-dependencies-
+                      - v2-dependencies-{{ checksum "package.json" }}
```

Comment:
This is fine. It won't really do anything except force a full install of dependencies (instead of from cache) the next time the job runs. This already happens any time the package.json changes since its hash is part of the key too.